app_dev.yaml

```diff
@@ -245,8 +245,21 @@ libraries:
 - name: webapp2
   version: "2.5.2"
 
+# This is an access connector that connects to the Google Cloud Redis instance
+# used for memory caching. Please replace PROJECT_ID with correct project id for
+# oppia on the production server.
+vpc_access_connector:
+  name: projects/PROJECT_ID/locations/REGION/connectors/oppia-redis-connector-1
+
 env_variables:
   PYTHONHTTPSVERIFY: 1
+# GAE_USE_SOCKETS_HTTPLIB is needed for the redis connector in the Python 2
+# runtime. More information can be found here:
+# https://cloud.google.com/appengine/docs/standard/python/connecting-vpc#configuring
+# (The second note under 'Configuring  your app to use a connector') which
+# redirects to these instructions to disable URL fetch:
+# https://cloud.google.com/appengine/docs/standard/python/sockets#making_httplib_use_sockets.
```

Comment:
Can you drop the period at the end, or at least put a space between it and the URL?